packages/fluentui/local-sandbox/package.json

```diff
@@ -10,7 +10,8 @@
     "serve": "node ./server.js"
   },
   "dependencies": {
-    "@fluentui/react-northstar": "^0.51.0"
+    "@fluentui/react-northstar": "^0.51.0",
+    "office-ui-fabric-react": "^7.124.0"
```

Comment:
I'm curious if this will cause any issues. (I'm not familiar with the local-sandbox package, @mnajdova @miroslavstastny or @layershifter should double check)

Reply:
The purpose of `local-sandbox` is to be to able to test some code locally with your current changes like in 3rd app.

I don't think that changes there and in `packages/fluentui/local-sandbox/src/index.tsx` are needed, I suggest to move them to E2E tests.